### PR TITLE
cleaned up test setup procedure

### PR DESCRIPTION
**Pull request by JDziewonski98**
# Summary
Cleaned up setup for tests by creating associations in only one file rather than in each,
also added an env variable to determine SQL logging (put SQL_LOGGING=false in your .env file if you don't want to see all the spam console output during tests)

## Test Plan
Ran the tests

## Related Issues
none

### CODE REVIEW

File: backend/src/models/tests/mural.test.ts [hunk 1]
```diff
@@ -19,25 +19,6 @@ const params = {
   },
 };
 
-beforeAll(async () => {
```
Comment (tedspare) [inline backend/src/models/tests/mural.test.ts:22]: what does it mean to remove the `beforeAll` associations throughout?

File: backend/src/testsetup/setupTests.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { database } from "../config/database";
+import { MuralCollection } from "../models/muralcollection.model";
+import { Tour } from "../models/tour.model";
+import { Mural } from "../models/mural.model";
+import { Borough } from "../models/borough.model";
+import { Artist } from "../models/artist.model";
+
+beforeAll(async () => {
```
Comment (tedspare) [inline backend/src/testsetup/setupTests.ts:8]: did you replace the individual `beforeAll`s with this one?

**Review by tedspare — APPROVED** [review]
100% of tests pass on my end. LGTM

File: backend/src/testsetup/setupTests.ts (hunk 2 above)
Comment (JDziewonski98) [reply backend/src/testsetup/setupTests.ts:8]: correct, the beforeAll/afterAll in this file run before and after every individual test suite. We had a beforeAll setting up the sequelize associations and an afterAll disconnecting from the database in basically every test suite, so it is way cleaner to put them all here.

**Pull request merged**